pkg/tool/config.go

```diff
@@ -3,24 +3,46 @@ package tool
 import (
 	"log"
 	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
 )
 
 type Config struct {
+	FS           afero.Fs
 	WorkingDir   string
+	RootDir      string
 	ManifestName string
 	BinDirName   string
 	Verbose      bool
 	Log          *log.Logger
 }
 
+func (c *Config) RequireManifest() error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Config.RequireManifest should have comment or be unexported

Reply:
fixed in 6f138e2